section1/reflection.md

```diff
@@ -2,18 +2,39 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+  I think I ask them too soon. I need to spend more time going back and making sure I've read everything to help me answer a question.
+
 2. What are the data types you learned about in this section? In your own words, define each.
 
+  *Number* a data type that handles all numbers, including negative numbers and decimals. It does not need quotes around it and no commas for numbers over 3 digits.
+
+  *String* data type consisting of letter and other characters, all wrapped in single or double quotes unless you are writing in interpolation, then a string would be wrapped in back tics along with the entire logged code.
+
+  *Boolean* Data type evaluating one of two values: true or false. helps us determine which script should run.
+
 3. How would you log the string `"Hello World!"` to the console?
 
+console.log("Hello World!");
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+// or /*The purpose of a code comment is to provide context and intent for yourself or another programmer in the future*/
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+  A variable is a lockbox of information that has a value. It's like tupperware. you can label it "leftovers" and now the chicken tikka masala is labeled "leftovers." When someone asks you what you are eating for dinner, you can say "leftovers" which is actually chicken tikka masala.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+ I use chess.com a lot. I think variables used could be dimensions, titles, scores. Numbers could be used in dimensions, letters could be uses in titles and Boolean could be used in calculating scores.
+
 7. In your own words, explain what concatenation is.
 
+  Concatenation is when you combine variables to log them in the console is a specific order.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+  Rosetta Stone might use concatenation to give different examples of phrases in other languages.
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+  I'm having difficulty thinking of how variables are used. I know that they are, but it's hard for me to grasp the real life uses for them outside of these exercise. so I guess my question is how are variable actually used?
```

Comment:
A common example could be getting a website visitor's name when they register on a website, and then displaying that name when they login.

Reply:
Okay this makes sense. Thanks for example!